bitnami/mongodb/values-production.yaml

```diff
@@ -331,7 +343,69 @@ customReadinessProbe: {}
 ##       - name: portname
 ##         containerPort: 1234
 ##
-initContainers: {}
+#initContainers: {}
+initContainers:
```

Comment:
I am not sure about why this is needed

Reply:
this initContainer is where the generation of the MongoDB server/client certs will be created during the cluster setup in order to let us access the cluster. You can also give it SAN's with this initContainer. Without the initContainer it will be impossible to start the cluster with the certs already generated ahead of time and you will not be able to access the cluster.

Reply:
Then I would add this to the statefulset and add a condition for `tls.enabled` instead of adding it to the `values.yaml`.
Also, in our charts we use bitnami images so please try to use bitnami/nginx instead.

Reply:
done